src/js/addons/pickers/Picker.js

```diff
@@ -54,11 +51,18 @@ export default class Picker {
         
         this.canvas = document.createElement('canvas');
         this.canvas.className = this.CSS_PREFIX + 'canvas picker-canvas';
-        this.canvas.width = this.width;
-        this.canvas.height = this.height;
+        // this.canvas.width = this.width;
+        // this.canvas.height = this.width;
 
         this.el.appendChild(this.canvas);
         this.ctx = this.canvas.getContext('2d');
+
+        // Sets canvas to the proper dimensions and at the correct pixel density
+        // Thanks Lou https://github.com/louh/bart/blob/gh-pages/main.js#L19-L23
```

Comment:
Thanks for the shoutout but this is pretty standard stuff :smiley: 
Not necessary to crowd the code with this!